### PR TITLE
docs: restructure quickstart 

### LINKED ISSUE #277: restructure quick start
**Describe the solution you'd like**
we should restructure quick start so it's not too long and doesn't contain feature descriptions (like exclusion). these features should be moved into their own pages

### PR DESCRIPTION
**Pull request by ashnamehrotra**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #277

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/docs/exclusion.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Excluding registries, repositories, and images
```
Comment (sozercan) [inline docs/docs/exclusion.md:5]: do we need to add this to sidebar?

Comment (ashnamehrotra) [reply docs/docs/exclusion.md:5]: im not sure how to test that, is it autogenerated?

Comment (sozercan) [reply docs/docs/exclusion.md:5]: ah looks like it's autogenerated https://github.com/Azure/eraser/blob/main/docs/sidebars.js#L17

but we need to define the position
https://github.com/Azure/eraser/blob/main/docs/docs/installation.md?plain=1#L3

can we create a parent item to this? perhaps "how to use eraser", "configuration" or something? or rename "quick start" to "how to use eraser"?

@salaxander any thoughts?

Comment (salaxander) [reply docs/docs/exclusion.md:5]: Yeah I've been thinking about how to break things down, I think having a parent item would be good

File: docs/docs/quickstart/_category_.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Quick Start",
```
Comment (sozercan) [inline docs/docs/quickstart/_category_.json]: do we want to change the name in this file and the directory name?

Comment (ashnamehrotra) [reply docs/docs/quickstart/_category_.json]: change the name to How to Use Eraser? should I change the how-to-use file back to quickstart also?

**Comment by salaxander** [comment]
If it's alright, I think what I'd like to do is structure this like Workload ID is doing (I can do another PR later if that's better)
```
docs/
├─ introduction.md
├─ installation.md
├─ quick-start.md (a true "quick" start guide with no configuration (probably using the scan flow)
├─ topics/
│  ├─ manual-removal.md
│  ├─ scanners.md
│  ├─ exclusions.md
│  ├─configuration.md
├─ development/
│  ├─ setup.md
│  ├─ releasing.md
```

How does this sound to folks? @sozercan

File: docs/docs/quick-start.md [hunk 3]
```diff
@@ -74,6 +74,8 @@ docker.io/library/alpine@sha256:8421d9a84432575381bfabd248f1eb56f3aa21d9d7cd2511
 
 ```
 
+<<<<<<< HEAD:docs/docs/quickstart/how-to-use.md
```
Comment (salaxander) [inline docs/docs/quick-start.md]: Looks like we've got some rebase lines here

File: docs/docs/exclusion.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Exclusion
+sidebar_position: 2
```
Comment (salaxander) [inline docs/docs/exclusion.md]: With the sidebar changes, you no longer have to have the `sidebar_position` item

File: docs/docs/quick-start.md [hunk 5]
```diff
@@ -74,6 +74,8 @@ docker.io/library/alpine@sha256:8421d9a84432575381bfabd248f1eb56f3aa21d9d7cd2511
...
+=======
 ## Excluding registries, repositories, and images
```
Comment (salaxander) [inline docs/docs/quick-start.md]: Should be able to remove this with the dedicated doc now

**Review by salaxander — CHANGES_REQUESTED** [review]

File: docs/docs/quick-start.md (hunk 3 above)
Comment (sozercan) [inline docs/docs/quick-start.md]: leftover?

File: docs/docs/quick-start.md [hunk 6]
```diff
@@ -88,6 +90,7 @@ EOF
 $ kubectl create configmap excluded --from-file=excluded=sample.json --namespace=eraser-system
 ```
 
+>>>>>>> 29b61ce3c26cfa33c86168831296fe018673fca9:docs/docs/quick-start.md
```
Comment (sozercan) [inline docs/docs/quick-start.md]: same

File: docs/docs/quick-start.md [hunk 7]
```diff
@@ -137,3 +140,67 @@ eraser-kind-worker-wfqc                      0/1     Completed   0          17s
...
+```shell
+$ docker exec kind-worker ctr -n k8s.io images list | grep alpine
+```
+
+If the image has been successfully removed, there will be no output.
+
+## Features
+* [Excluding registries, repositories, and images](exclusion.md) from removal
+=======
+>>>>>>> 29b61ce3c26cfa33c86168831296fe018673fca9:docs/docs/quick-start.md
```
Comment (sozercan) [inline docs/docs/quick-start.md]: same

File: docs/docs/quick-start.md [hunk 8]
```diff
@@ -137,3 +140,67 @@ eraser-kind-worker-wfqc                      0/1     Completed   0          17s
 eraser-kind-worker2-gwbit                    0/1     Completed   0          17s
 eraser-scanner-78p49-vxb4j                   0/1     Completed   0          30s
 ```
+<<<<<<< HEAD:docs/docs/quickstart/how-to-use.md
```
Comment (sozercan) [inline docs/docs/quick-start.md]: same

**Review by sozercan — CHANGES_REQUESTED** [review]
a few merge conflict leftovers

File: docs/docs/quick-start.md (hunk 3 above)
Comment (ashnamehrotra) [reply docs/docs/quick-start.md]: sorry I missed that!

**Review by sozercan — APPROVED** [review]

**Comment by sozercan** [comment]
@salaxander lgty?

**Review by salaxander — APPROVED** [review]
LGTM

**Pull request merged**